.circleci/common.yml

```diff
@@ -162,16 +162,16 @@ commands:
       - run:
           name: Install pnpm package manager
           command: |
-            corepack prepare pnpm@8.15.6 --activate
-            pnpm config set store-dir .pnpm-store
+            corepack prepare pnpm@9.1.1 --activate
+            corepack pnpm config set store-dir .pnpm-store
```

Comment:
... so.. since we use the CircleCI  node docker image which has a version of pnpm installed that we can not override, we need to prefix all usages of PNPM with corepack (for the execution contexts that run in CirlceCI node docker)